src/performance/longtask.ts

```diff
@@ -0,0 +1,14 @@
+import { Event } from "@clarity-types/data";
+import { LongTaskState } from "@clarity-types/performance";
+import encode from "./encode";
+
+export let state: LongTaskState = null;
+
+export function reset(): void {
+    state = null;
+}
+
+export function compute(entry: PerformanceEntry): void {
+    state = { time: Math.round(entry.startTime), data: { duration: Math.round(entry.duration) } };
```

Comment:
Add attribution property. See [https://w3c.github.io/longtasks/#sec-PerformanceLongTaskTiming](https://w3c.github.io/longtasks/#sec-PerformanceLongTaskTiming)

Reply:
In practice I wasn't able to see valid values in attribution (self) & (multiple). However, I agree may be should add it and review data at scale. Will add. 

---
In reply to: [347589413](https://github.com/microsoft/clarity-js/pull/160#discussion_r347589413) [](ancestors = 347589413)